src/ui/targetingTree.ts

```diff
@@ -82,14 +82,18 @@ const insertDefinitionTree = (rootTree: Tree, audience: Audience) => {
             }
             userFilter.insert(comparatorMap[filter.comparator], values)
             const userProperty = filter.subType === 'customData' ? filter.dataKey : subTypeMap[filter.subType]
-            definitionTree.insert(userProperty, userFilter)
+            insertWithPrefixAND(definitionTree, userProperty, index, userFilter)
```

Comment:
since the tree and value aren't relevant to the prefixing, we could just pass the string and index to this helper
```suggestion
            const prefixedProperty = prefixWithAnd(userProperty, index)
            definitionTree.insert(prefixedProperty, userFilter)
```